addons/project/static/tests/project_subtask_kanban_list_form_tests.js

```diff
@@ -146,8 +148,25 @@ QUnit.module('Subtask Kanban List tests', {
         await click(target.querySelector(".o_field_x2many_list_row_add a"));
         await clickDropdown(target, 'project_id');
         await clickOpenedDropdownItem(target, 'project_id', 'Project One');
+        await editInput(target, ".o_data_row > td[name='name'] > div > input", "aaa");
         await click(target, ".o_form_button_save");
         assert.equal(target.querySelector('.o_field_project').textContent.trim(), 'Project One')
     });
 
+    QUnit.test("focus new subtask's name", async function (assert) {
```

Comment:
It seems the test fails in the runbot, can you check?

Reply:
The cause of the failed test is to replace isDirty with Dirty in this commit- [link](https://github.com/odoo/odoo/commit/8723f020c3587a900c811b8cc23f53fe34b98df3)